src/olympia/search/tests/test_views.py

```diff
@@ -1,1048 +0,0 @@
-        response = self.client.get(self.url, {'appver': '4.0'})
-        assert self.get_results(response) == []
-
-        # Search for add-ons compatible with Firefox 10.0: all should be found.
-        response = self.client.get(self.url, {'appver': '10.0'})
-        expected_addons_pks = sorted(self.addons.values_list('id', flat=True))
-        assert self.get_results(response) == expected_addons_pks
-
-        # Set strict compatibility to True on one of them, it should no longer
-        # be returned when searching for add-ons compatible with 10.0 since the
-        # max version is 5.0.99.
-        addon = self.addons[0]
-        addon.current_version.files.update(strict_compatibility=True)
-        addon.save()
-        self.refresh()
-        response = self.client.get(self.url, {'appver': '10.0'})
-        expected_addons_pks = sorted(
-            self.addons.exclude(pk=addon.pk).values_list('id', flat=True))
-        assert self.get_results(response) == expected_addons_pks
-
-    def test_results_platform_filter_all(self):
-        for platform in ('', 'all'):
-            response = self.client.get(self.url, {'platform': platform})
-            assert self.get_results(response) == (
-                sorted(self.addons.values_list('id', flat=True)))
-
-    def test_slug_indexed_but_not_searched(self):
-        addon = self.addons[0]
-
-        response = self.client.get(self.url, {'q': 'omgyes'})
-        assert self.get_results(response) == []
-
-        addon.update(slug='omgyes')
-        self.refresh()
-        response = self.client.get(self.url, {'q': 'omgyes'})
-        assert self.get_results(response) == []
-
-    def test_authors_indexed_but_not_searched(self):
-        addon = self.addons[0]
-
-        response = self.client.get(self.url, {'q': 'boop'})
-        assert self.get_results(response) == []
-
-        AddonUser.objects.create(
-            addon=addon, user=UserProfile.objects.create(username='boop'))
-        AddonUser.objects.create(
-            addon=addon, user=UserProfile.objects.create(username='ponypet'))
-        addon.save()
-        self.refresh()
-        response = self.client.get(self.url, {'q': 'garbage'})
-        assert self.get_results(response) == []
-        response = self.client.get(self.url, {'q': 'boop'})
-        assert self.get_results(response) == []
-        response = self.client.get(self.url, {'q': ''})
-        del addon
-        assert self.get_results(response) == [
-            addon.id for addon in self.addons]
-
-    def test_search_doesnt_return_unlisted_addons(self):
-        addon = self.addons[0]
-        response = self.client.get(self.url, {'q': 'Addon'})
-        assert addon.pk in self.get_results(response)
-
-        self.make_addon_unlisted(addon)
-        addon.reload()
-        self.refresh()
-        response = self.client.get(self.url, {'q': 'Addon'})
-        assert addon.pk not in self.get_results(response)
-
-    def test_find_addon_default_non_en_us(self):
```

Comment:
This test might be useful to keep and move elsewhere. I don't think we have an equivalent in the API tests

Reply:
I think this is covered in https://github.com/mozilla/addons-server/blob/master/src/olympia/addons/tests/test_views.py#L2937:L2973 (with less comments)